proxy/core/acceptor/acceptor.py

```diff
@@ -117,8 +134,15 @@ def run_once(self) -> None:
                         if mask & selectors.EVENT_READ:
                             if self.sock is not None:
                                 conn, addr = self.sock.accept()
-                                addr = None if addr == '' else addr
-                                self._work(conn, addr)
+                                logging.debug(
```

Comment:
The same comment as in https://github.com/abhinavsingh/proxy.py/pull/769#discussion_r753842787 applies here (and in any similar places).